doc-content/enterprise-only/release-notes/rn-731_fixed-issues-ref.adoc

```diff
@@ -0,0 +1,85 @@
+
+== Form modeler
+* When you start a process instance in {CENTRAL} the modal window contains the previously rendered form until the new form renders [https://issues.jboss.org/browse/RHPAM-1951[RHPAM-1951]]
+* The multiple selector throws an error when edited [https://issues.jboss.org/browse/RHPAM-1861[RHPAM-1861]]
+
+== Installer
+* The default placeholder for the controller URL should contain `{URL_COMPONENT_CENTRAL}` when installing {KIE_SERVER} and {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1774[RHPAM-1774]]
+* The controller URL is not updated in XML files when only {KIE_SERVER} is being installed [https://issues.jboss.org/browse/RHPAM-1781[RHPAM-1781]]
+* You cannot use the installer to install {PRODUCT} on JWS 5.1 [https://issues.jboss.org/browse/RHPAM-2038[RHPAM-2038]]
+
+== OpenShift
+
+* Setting the Maven mirror URL does not exclude {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1994[RHPAM-1994]]
+* The KIE server image does not run correctly when EJB is initialized first [https://issues.jboss.org/browse/RHPAM-1970[RHPAM-1970]]
+* {CENTRAL} throws an exception when the user clicks a deployment unit [https://issues.jboss.org/browse/RHPAM-1952[RHPAM-1952]]
+* `ServerTemplateConverter` breaks with `KieServerMode` mixed-case [https://issues.jboss.org/browse/RHPAM-1975[RHPAM-1975]]
+
+== Process designer
+
+* You cannot open a callable process that contains spaces and extended chars [https://issues.jboss.org/browse/RHPAM-2006[RHPAM-2006]]
+
+
+== Process engine
+
+* {KIE_SERVER} is not removed from the controller view if the server is killed or crashes [https://issues.jboss.org/browse/RHPAM-1654[RHPAM-1654]]
+* The newly added `org.jbpm.correlationkey.length` system property does not truncate the *Value* column in the `CorrelationPropertyInfo` table [https://issues.jboss.org/browse/RHPAM-1961[RHPAM-1961]]
+* The {KIE_SERVER} client does not reuse the marshaller for JMS [https://issues.jboss.org/browse/RHPAM-1988[RHPAM-1988]]
+* An EJB timer error occurs in development mode for an existing process during deploy [https://issues.jboss.org/browse/RHPAM-1985[RHPAM-1985]]
+
+//== PMML
+
+//* `ScoringStrategiesTest`, `ScorecardProviderTest`, `ExternalObjectModelTest`, `ScoreCardProviderPMMLTest`, and `ScoreCardReasonCodeTest` fail on Windows [https://issues.jboss.org/browse/RHPAM-1815[RHPAM-1815]]
+
+
+endif::[]
+
+ifdef::DM[]
+
+
+== Installer
+
+* When installing only Decision Server, the installer does not create the user specified on the user creation screen [https://issues.jboss.org/browse/RHDM-885[RHDM-885]]
+
+
+== {CENTRAL}
+
+* If you add an action value for a particular guided decision table row, the value is also applied to other rows  [https://issues.jboss.org/browse/RHDM-872[RHDM-872]]
+* Manual changes to the `kmodule.xml` file are overridden [https://issues.jboss.org/browse/RHDM-599[RHDM-599]]
```

Comment:
Minor question:

Do you think this line and linked Jira should use "overwritten" rather than "overridden" to describe what is happening? It appears the file state is reverted, meaning the changes to the file are overwritten, rather than the new configuration being overridden by the existing configuration.